toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -29,6 +31,28 @@ import (
 	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/timestamp"
 )
 
+var (
+	// Every valid line will be of the form: <package_name>.<architecture> <version>.<dist> <repo_id>
```

Comment:
This is slightly different from the one in `rpmrepocloner,go`, can we unify them both into one spot and make it part of the `tdnf` package maybe, along with the calling code into a function?

Reply:
Since this is tied to a P0 bug, I'm trying to get it merged ASAP. Unless the refactoring is quick and simple, I'd prefer to merge this as is. Perhaps I can file a follow-up task for the refactoring. Let me know if you have a minute to discuss.

Reply:
+1 on moving both regular expressions and their corresponding indices constants into `tdnf.go`, as that should be a small change having little risk of breaking anything else. I have no strong opinions on extracting or writing any additional functions around that.

Reply:
Sure, I've moved these both to tdnf.go